payments-core/src/main/java/com/stripe/android/payments/core/injection/Stripe3DSAuthenticatorModule.kt

```diff
@@ -1,29 +1,25 @@
 package com.stripe.android.payments.core.injection
 
-import android.content.Context
 import com.stripe.android.PaymentAuthConfig
 import com.stripe.android.model.StripeIntent
 import com.stripe.android.model.StripeIntent.NextActionData
 import com.stripe.android.payments.core.authentication.PaymentAuthenticator
-import com.stripe.android.payments.core.authentication.threeds2.DefaultStripe3ds2ChallengeResultProcessor
 import com.stripe.android.payments.core.authentication.threeds2.Stripe3DS2Authenticator
-import com.stripe.android.payments.core.authentication.threeds2.Stripe3ds2ChallengeResultProcessor
-import com.stripe.android.stripe3ds2.service.StripeThreeDs2Service
-import com.stripe.android.stripe3ds2.service.StripeThreeDs2ServiceImpl
-import com.stripe.android.stripe3ds2.transaction.MessageVersionRegistry
 import dagger.Binds
 import dagger.Module
 import dagger.Provides
 import dagger.multibindings.IntoMap
-import javax.inject.Named
 import javax.inject.Singleton
-import kotlin.coroutines.CoroutineContext
 
 /**
  * Provides [PaymentAuthenticator] for [NextActionData.SdkData.Use3DS2].
  * Requires 3ds2 SDK.
  */
-@Module
+@Module(
+    includes = [
+        Stripe3ds2TransactionViewModelModule::class
+    ]
+)
 internal abstract class Stripe3DSAuthenticatorModule {
```

Comment:
This is the "bigger" graph, it's responsible for `Stripe3ds2Authenticator` and `Stripe3ds2TransactionViewModelModule`.

When being recovered from a killed process, we don't need to recreate `Stripe3ds2Authenticator`, hence the "smaller" graph is enough